orc8r/cloud/helm/orc8r/charts/secrets/templates/configs-orc8r.secret.yaml

```diff
@@ -21,7 +21,21 @@ metadata:
 {{ include "labels" . | indent 4 }}
 data:
 {{- if .Values.secret.configs.enabled }}
+{{ $thanos_enabled := .Values.thanos_enabled}}
 {{- range $key, $value := .Values.secret.configs.orc8r }}
+  # Set metricsd.yml explicitly if thanos is enabled
+  {{- if eq $key "metricsd.yml" }}
+    {{- if $thanos_enabled }}
+  {{ $value = `
+       profile: "prometheus"
+
+       prometheusQueryAddress: "http://orc8r-thanos-query-http:10902"
```

Comment:
template in the deployment name instead of harcoding orc8r, unless these names are hardcoded with orc8r- prefix. Lots of other components use the deployment name as the prefix